docker/nwchem/src/run.py

```diff
@@ -0,0 +1,83 @@
+import os
+import subprocess
+import jinja2
+import json
+
+import openchemistry as oc
+
+def run_calculation(geometry_file, output_file, params, scratch_dir):
+    # Read in the geometry from the geometry file
+    # This container expects the geometry file to be in .xyz format
+    with open(geometry_file) as f:
+        xyz_structure = f.read()
+        # remove the first two lines in the xyz file
+        # (i.e. number of atom and optional comment)
+        xyz_structure = xyz_structure.split('\n')[2:]
+        xyz_structure = '\n  '.join(xyz_structure)
+
+    # Read the input parameters
+    theory = params.get('theory', 'hf')
+    task = params.get('task', 'energy')
+    basis = params.get('basis', 'cc-pvdz')
+    functional = params.get('functional', 'b3lyp')
+    charge = params.get('charge', 0)
+    multiplicity = params.get('multiplicity', 1)
+
+    if theory.lower() == 'ks':
+        _theory = functional
+    else:
+        _theory = 'scf'
+
+    reference = theory.lower()
+
+    if multiplicity == 1:
+        reference = 'r' + reference
+    else:
+        reference = 'u' + reference
+
+    optimization = params.get('optimization', None)
+    vibrational = params.get('vibrational', None)
+    charge = params.get('charge', 0)
+    multiplicity = params.get('multiplicity', 1)
+    theory = params.get('theory', 'scf')
+    functional = params.get('functional', 'b3lyp')
+    basis = params.get('basis', 'cc-pvdz')
```

Comment:
Are these default right for NWChem?

Reply:
The Hartree-Fock task is also called `scf` so that is fine. Singlet spin state multiplicity is `1` so that is ok, too. The functional variable is just used when `theory = dft`. The basis set name is standard so that should be fine. I will commit a new change that will handle dft and hf.